flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateTable.java

```diff
@@ -237,6 +239,36 @@ public String getColumnSqlString() {
 		return writer.toString();
 	}
 
+	/**
+	 * Split the computed columns expression into string key-value pairs.
+	 *
+	 * <p>For example, {@code col2 as to_timestamp(col1)} would be split into pair:
+	 * (col2, to_timestamp(col1)).
+	 *
+	 * @return computed column name to expression mapping
+	 **/
+	public Map<String, String> getComputedColumnMap() {
```

Comment:
This is never used.

Reply:
Would be used in the following PR, i prefer to keep it.

Reply:
I would prefer to add it in your following PR. There is no "context" for reviewers no judge whether this method is reasonable to add.

Reply:
+1 to introduce it when we need it.

Reply:
Removed